packages/hash/frontend/src/pages/[shortname]/types/entity-type/[entity-type-id].page/definition-tab/property-list-card/property-expected-values.tsx

```diff
@@ -1,36 +1,58 @@
 import { PropertyType } from "@blockprotocol/type-system";
-import { Chip } from "@hashintel/hash-design-system";
-import { types } from "@hashintel/hash-shared/ontology-types";
+import { Chip, FontAwesomeIcon } from "@hashintel/hash-design-system";
+import { chipClasses } from "@mui/material";
+import { Stack } from "@mui/system";
 
-const dataTypeIdToTitle = Object.values(types.dataType).reduce<
-  Record<string, string>
->((prev, dataType) => {
-  const { dataTypeId, title } = dataType;
-
-  return { ...prev, [dataTypeId]: title };
-}, {});
+import { expectedValuesOptions } from "./shared/expected-values-options";
+import { getArrayExpectedValueType } from "./shared/get-expected-value-descriptor";
 
 // @todo handle this being too many
```

Comment:
This you've handled this todo?

Reply:
Not sure what the comment is about. You added it, do you remember what it was?

Reply:
It was just about it appearing properly when there were too many types to fit on one row, which I think you've done now